blogposts/2024-07-18-running-rnw-from-a-mac.md

```diff
@@ -0,0 +1,86 @@
+
+The React Native for Windows team has heard the community feedback. While running on a Windows device will be the best experience for developing on Windows, we recognize that not everyone involved in a development team can have access to multiple devices to develop for different operating systems. We also see that MacBook computers are used widely within the React Native developer community.
+
+In this blog post, I'm going to walk through a few options for how to develop React Native for Windows applications from your MacBook computer because owning a Windows device is not a requirement to build and ship Windows experiences.
+
+To get started building Windows experiences from a non-Windows device, you can leverage virtual machine software. There are several options you have as a developer to spin-up a Windows VM from a Mac computer including [Parallels](https://www.parallels.com/), [VirtualBox](https://www.virtualbox.org/), [VMWare](https://www.vmware.com/), and [Microsoft Dev Box](https://azure.microsoft.com/products/dev-box/) to name a few. In this post, I'll walk through my experience using Parallels and Microsoft Dev Box to build React Native for Windows applications from a MacBook.
+
+## Creating a Windows VM using Parallels
+
+One option to create VMs on macOS is [Parallels](https://www.parallels.com/). Parallels is a software which can be locally installed on a developer's MacOS machine to be used to create virtual machines of a variety of operating systems.
+
+After signing up for the [Parallels free trial](https://www.parallels.com/products/desktop/trial/), you will be given a download of the Parallels Desktop software and installation instructions. After Parallels Desktop was installed and running, the software will prompt you to install Windows 11. From there, your Windows VM is ready to be run.
+
+With my virtual machine running, I was able to use the VM as if I was on a Windows computer. From here, I could follow the ["System Requirements"](https://microsoft.github.io/react-native-windows/docs/rnw-dependencies) and ["Getting Started"](https://microsoft.github.io/react-native-windows/docs/getting-started) documentation for React Native for Windows. Parallels had access to my local files, so I did not need to redownload any source code to get started developing. When I was finished with my development session, I was able to close the virtual machine and the machine's state was saved for next time.
+
+![](assets/2024-07-18-running-rnw-from-a-mac/Screenshot-Parallels.jpg)
+
+## Takeaways about Parallels
+
+In less than an hour, I was able to spin up a Windows VM on a Mac and use that environment to build a React Native for Windows app. Windows VM’s allowed me to quickly experiment with adding Windows support to my existing React Native app because of the low level of overhead to get a VM up and running.
```

Comment:
```suggestion
In less than an hour, I was able to spin up a Windows VM on my Mac and use that environment to build a React Native for Windows app. Windows VMs allowed me to quickly experiment with adding Windows support to my existing React Native app because of the low level of overhead to get a VM up and running.
```